assets/js/modules/analytics/components/settings/SettingsForm.stories.js

```diff
@@ -343,6 +343,13 @@ WithDashboardViewToggle.decorators = [
 			registry
 				.dispatch( MODULES_ANALYTICS )
 				.setDashboardView( DASHBOARD_VIEW_GA4 );
+
+			registry.dispatch( MODULES_ANALYTICS ).selectProperty(
+				properties[ 0 ].id,
+				// eslint-disable-next-line sitekit/acronym-case
+				properties[ 0 ].internalWebPropertyId
+			);
```

Comment:
Since we're no longer auto-selecting a UA property and the stories here expect one to be auto-selected due to the `selectAccount()` dispatch, we're manually selecting a property here.